src/main/java/edu/cornell/kfs/ksr/KSRKeyConstants.java

```diff
@@ -23,6 +23,7 @@ public class KSRKeyConstants {
     public static final String ERROR_SECURITY_REQUEST_DOC_DEPENDENT_ROLE_MISSING = "error.ksr.securityrequestdocument.dependentRole.missing";
     public static final String ERROR_SECURITY_REQUEST_DOC_CHANGE_MISSING = "error.ksr.securityrequestdocument.change.missing";
     public static final String ERROR_SECURITY_REQUEST_DOC_SERVICE_EXCEPTION = "error.ksr.securityrequestdocument.service.exception";
+    public static final String ERROR_SECURITY_REQUEST_DOC_PRIMARY_DEPARTMENT = "error.ksr.securityrequestdocument.service.primary.department";
```

Comment:
This new constant doesn't seem to be used by this code at all, and I don't think it's been added to the app resource properties either. Can it be removed?